packages/mesos/buildinfo.json

```diff
@@ -3,8 +3,8 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/mesosphere/mesos",
-    "ref": "33240d706765deb857bcc8b0b65b50d302aa17c9",
-    "ref_origin" : "dcos-mesos-1.4.x-e185d805"
+    "ref": "b0fd885dbf02a61b5635184df19905695b4bba96",
+    "ref_origin" : "dcos-mesos-1.4.0-rc1"
```

Comment:
If you're adding three more cherry-picks to this branch from master, please update the README so we don't lose them when we create an rc2 branch.
Or, if it's important enough and low-risk enough, talk to @karya0 and @hatred about landing the patches in 1.4.x proper so they're automatically included in 1.4.0-rc2 or 1.4.1, whichever's next.